src/main/java/com/exactprosystems/remotehand/web/actions/WaitForChanges.java

```diff
@@ -80,7 +78,7 @@ public String run(WebDriver webDriver, By webLocator, Map<String, String> params
 		do
 		{
 			WebElement element = findElement(webDriver, webLocator);
-			byte[] currentState = ScreenshotRegionUtils.takeElementScreenshot(webDriver, element);
+			byte[] currentState = screenWriter.takeElementScreenshot(webDriver, element);

```

Comment:
This code should work only with png.
Encoding screenshot to webp can take a long time.
Code is in loop so there are many iterations